jumanji/environments/logic/minesweeper/env_test.py

```diff
@@ -101,17 +101,17 @@ def test_default_reward_and_done_signals(
 
 def test_minesweeper_env_reset(minesweeper_env: Minesweeper) -> None:
     """Validates the jitted reset of the environment."""
-    reset_fn = jit(minesweeper_env.reset)
-    key = random.PRNGKey(0)
+    reset_fn = jax.jit(minesweeper_env.reset)
+    key = jax.random.PRNGKey(0)
     state, timestep = reset_fn(key)
     assert isinstance(timestep, TimeStep)
     assert isinstance(state, State)
     assert state.step_count == 0
-    assert state.flat_mine_locations.shape == (minesweeper_env.num_mines,)
-    assert timestep.observation.num_mines == minesweeper_env.num_mines
+    assert state.flat_mine_locations.shape == (minesweeper_env.generator.num_mines,)
+    assert timestep.observation.num_mines == minesweeper_env.generator.num_mines
     assert state.board.shape == (
-        minesweeper_env.num_rows,
-        minesweeper_env.num_cols,
+        minesweeper_env.generator.num_rows,
+        minesweeper_env.generator.num_cols,
```

Comment:
After doing the previous change:
```suggestion
        minesweeper_env.num_rows,
        minesweeper_env.num_cols,
```